saleor/graphql/account/mutations/staff.py

```diff
@@ -369,6 +370,14 @@ def perform_mutation(cls, _root, info, address_id, user_id, **data):
         return cls(user=user)
 
 
+class AddressUpdate(BaseAddressUpdate):
```

Comment:
Maybe we could name it `CustomerAddressUpdate`? So users can more easily assume this is a staff mutation.

Reply:
This is a good idea for the future. Now we can't use prefix "Customer" for all address mutation for staff user because in current API we have `CustomerAddressCreate` as a deprecated mutation for creating address by a logged-in user.